src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -10,18 +10,24 @@
 #include <uct/base/uct_cm.h>
 #include <ucs/datastruct/khash.h>
 #include <ucs/sys/string.h>
+#include <ucs/datastruct/bitmap.h>
+#include <uct/ib/mlx5/ib_mlx5.h>
 
 #include <rdma/rdma_cma.h>
 
 
-KHASH_MAP_INIT_INT64(uct_rdmacm_cm_cqs, struct ibv_cq*);
+KHASH_MAP_INIT_INT64(uct_rdmacm_cm_qp_contexts, struct uct_rdmacm_cm_qp_context*);
 
 
 #define UCT_RDMACM_TCP_PRIV_DATA_LEN    56    /** See rdma_connect(3) */
 #define UCT_RDMACM_EP_FLAGS_STRING_LEN  128   /** A string to hold the
                                                   representation of the ep flags */
 #define UCT_RDMACM_EP_STRING_LEN        192   /** A string to hold the ep info */
 
+/** The size(bits) of a reserved qpn block bitmap */
+#define UCT_RDMACM_RESERVED_QPN_BLK_BMP_BITS \
```

Comment:
maybe define a constant (256) and use it to define bitmap size `ucs_bitmap_t(UCT_RDMACM_RESERVED_QPN_BLK_BITMAP_SIZE)`

Reply:
@yosefe Removed bitmap.